### DISCUSSION TITLE
Group play

### DISCUSSION THREAD

**Discussion by maustru** [hklages/node-red-contrib-sonos-plus#283] [Q&A] [answered]
Hello together

okay, I think I'm too incompetent as I can't be alone in having this problem.
I have 3 Sonos speakers in the basement. When I turn them on as shown in the picture below, they start playing, but not 100% synchronised, as they get the signal individually and not as a group. In the Sonos APP I have the group "sonos-UG", then it works perfectly.
I can't enter multiple IP addresses for multiple speakers in the node either. 

![grafik](https://github.com/hklages/node-red-contrib-sonos-plus/assets/121907657/a41eff09-1043-4da5-8252-3d51792c7db9)

So how does this work?
Thanks and greetings

**Comment by hklages** [comment, marked as answer]
Good morning

Here is the documentation:
https://github.com/hklages/node-red-contrib-sonos-plus/wiki

Easiest way ist to group the player with "household.create.group" and then
just send a "group.play" to one of the player. This package automatically
routes the play command to the coordinator of the group (group. Commands).

Your group definition is just a definition. It also has to be activated.
Unfortunately activation of these group definitions is not possible with
this package.

Does that help?

**Comment by maustru** [comment]
Yes! Thanks a lot!
For everyone else who may be facing the same problem as me:
The Inject must pe look like this (example)
![grafik](https://github.com/hklages/node-red-contrib-sonos-plus/assets/121907657/54ad2058-f911-4ddf-9b96-e24880881c18)

create a "universal" node with the topic "household.create.group". In the "Config Node" you can take one of your Sonos player, with the exactly same name from the Sonos app, not your name, if it is different, .

**Comment by hklages** [comment]
Great. Thanks for the Feedback.
